### ISSUE TITLE
String pattern correction

### ISSUE THREAD

**Issue by svenbiellmann** [swissfintechinnovations/ca-payment#127]
### Contact Details

sven.biellmann@finnova.com

### Issue category

Bugfix

### Issue description

Certain string patterns in our API definitions do not comply with OpenAPI Specification definitions (single quotes and start and end delimiter are missing). 

### Issue reasoning

Correct sting pattern definitions to comply with OpenAPI Specification definitions (see also [https://swagger.io/docs/specification/data-models/data-types/](https://swagger.io/docs/specification/data-models/data-types/)).

### Request

Correct string patterns according to OpenAPI Specification definitions.

### Proposed change

```oasv2-yaml
Examples AS-IS
pattern: camt\.053\.[0-9]{3}\.[0-9]{2}
pattern: '[A-Z]{2,2}[0-9]{2,2}[a-zA-Z0-9]{1,30}'

========
Example TO-BE
pattern: '^camt\.053\.[0-9]{3}\.[0-9]{2}$'
pattern: '^[A-Z]{2,2}[0-9]{2,2}[a-zA-Z0-9]{1,30}$'
```

### Affected API's

Both

### Code of Conduct

- [X] I agree to follow this project's Contribution Guidelines

**Comment by svenbiellmann** [comment]
### Metting Notes 26.9.24

Just documentary issue. Annotation of pattern changed in order to be compliant with OpenAPI specification. Pull Request will be prepared by @svenbiellmann.

**Comment by svenbiellmann** [comment]
### Meeting Notes 31.10.24

The Pull Request has been approved and will be merged into V4.1.1.
